CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+New contributors are always welcome!
+
+Need to contact other collaborators in this project? Feel free to [join our slack group!](https://join.slack.com/t/githubdashboard/shared_invite/enQtNDcxMTM5OTMyNjExLWFmYTE1NTFiMzkyMzU0ZmRjMjI0YjI1OTVkMDk0MTUyZmJlMjM2NGUzODQ1YjZmZDVkMzkxYzgzYjM1MjI1ZGI)
+
+## Getting Started
+
+[Fork](https://help.github.com/articles/fork-a-repo/) and [clone](https://help.github.com/articles/cloning-a-repository/) the repository onto your computer.
+
+[Generate a github API access_token](https://github.com/settings/tokens) with repo permissions (but generate a new key with more permissions if the situation calls for it) and put that key in `config.json`. Install node.js if you haven't already, then run the following commands in the project directory to get the app working locally on your machine:
+
+```
+npm install
+npm start
+```
+
+This will build the project and automatically start the development server on [http://localhost:3000](http://localhost:3000). You wil now have a running copy of the project on your local machine.
+
+## Issues
+
+If you would like to start working on an issue, you can comment on it to claim it if nobody else has started working on it yet. Feel free to ask any questions on the issue page or in the [slack channel](https://join.slack.com/t/githubdashboard/shared_invite/enQtNDcxMTM5OTMyNjExLWFmYTE1NTFiMzkyMzU0ZmRjMjI0YjI1OTVkMDk0MTUyZmJlMjM2NGUzODQ1YjZmZDVkMzkxYzgzYjM1MjI1ZGI).
+
```

Comment:
If you're looking for a good starter bug, you can look for issues tagged with [good first issue](https://github.com/deepanjali19/GitHub-Dashboard/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).